pkgs/by-name/se/seclists/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "seclists";
+  version = "unstable-2023-08-15";
+
+  src = fetchFromGitHub {
+    owner = "danielmiessler";
+    repo = "SecLists";
+    rev = "395c945627196b6172e6e83a014c964f805e8d48";
+    hash = "sha256-dR48TxMGjUU++J4I0q7dL79LJJ53dh8IASp+inGKnmg=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/wordlists/seclists
+    cp -R Discovery Fuzzing IOCs Miscellaneous Passwords Pattern-Matching Payloads Usernames Web-Shells \
+      $out/share/wordlists/seclists
```

If you don't want to maintain the list of word lists here, you could match on directories that look like a word list.

```suggestion
    find . -maxdepth 1 -type d -regextype posix-extended -regex '^./[A-Z].*' -exec cp -R {} $out/share/wordlists/seclists \;
```